examples/user_profile.py

```
@@ -0,0 +1,96 @@
+
+# -----------------------------------------------------------------------------
+# Paradigm 1: Grouped Values
+# -----------------------------------------------------------------------------
+# We store a Python dictionary as the value for the sender's address.
+# Pros: Keeps all user data neatly organized in one object.
+# Cons: To update a single field, we must read the whole dict and write it back.
+
+if payload.startswith('set_grouped:'):
+    parts = payload.split(':')
+    if len(parts) != 3:
+        raise Exception("Invalid format. Use set_grouped:<age>:<name>")
```

Ruff (0.15.21)

[warning] Create your own exception

(TRY002)